release.config.mjs

```diff
@@ -26,15 +25,21 @@ export default {
 		[
 			'@semantic-release/exec',
 			{
-				prepareCmd:
-					'VERSION=${nextRelease.version} npx nx run-many -t release --parallel=1 && VERSION=${nextRelease.version} npx -p replace-json-property rjp ./package.json version ${nextRelease.version}',
+				prepareCmd: 'TAG=latest,VERSION=${nextRelease.version} pnpm run pre-release',
+				releaseCmd: 'TAG=latest,VERSION=${nextRelease.version} pnpm run release',
 			},
 		],
 		[
 			'@semantic-release/git',
 			{
-				assets: ['libs/**/package.json', 'package.json', 'CHANGELOG.md'],
-				message: 'chore(release): -v${nextRelease.version} [skip ci]\n\n${nextRelease.notes}',
+				assets: [
```

Comment:
is the root package.json not changed anymore?

Reply:
It never was, but we could store the current version as a source of truth there. I think keeping CLI and UI in sync also reduces additional complexity as we move forward